test/kitchen/site-cookbooks/dd-agent-step-by-step/recipes/default.rb

```diff
@@ -39,6 +39,15 @@
   end
 
 when 'rhel'
+  if platform?('centos') && node['dd-agent-rspec'] && node['dd-agent-rspec']['enable_cws']
+    # TODO(lebauce): enable repositories to install package
+    # package 'policycoreutils-python'
+
+    execute 'disable SElinux to be able to start system-probe' do
```

Comment:
```suggestion
    execute 'put SElinux in permissive mode to be able to start system-probe' do
```
The doc is making me think disabling SElinux and putting it to permissive mode is different, not sure though (see [here](https://man7.org/linux/man-pages/man8/setenforce.8.html) and [here](https://linux.die.net/man/8/setenforce)). Otherwise LGTM !

Reply:
From the [man page](https://linux.die.net/man/8/selinux):
```
The SELINUX variable may be set to any one of disabled, permissive, or enforcing to select one of these options. The disabled option completely disables the SELinux kernel and application code, leaving the system running without any SELinux protection. The permissive option enables the SELinux code, but causes it to operate in a mode where accesses that would be denied by policy are permitted but audited.
```

Reply:
It's not exactly the same thing, but for the purpose of these tests it makes no difference. It's also the same comment as the PR that initially introduced it, I just copy-pasted it. If we want to update, we should do it in all places where this is defined.